App/Sagas/NodeStarted.ts

```diff
@@ -0,0 +1,27 @@
+import { all, call, put, take } from 'redux-saga/effects'
+import { getType } from 'typesafe-actions'
+import TextileNode from '../../TextileNode'
+import AuthActions from '../Redux/AuthRedux'
+import PreferencesActions from '../Redux/PreferencesRedux'
+import TextileNodeActions from '../Redux/TextileNodeRedux'
+import ThreadsActions from '../Redux/ThreadsRedux'
+
+export function * onNodeStarted () {
```

Comment:
Wire up all things that happen once the node is started. 

Possible optimization: Split this into two functions, one for node started in background and one for node started in foreground. That would allow us to only do UI related things in the node has started and we're in the foreground.

Reply:
Separating them out makes sense to me @asutula. Certainly keeps things cleaner?

Reply:
Will do a a future revision